lib/wit.rb

```diff
@@ -34,9 +34,11 @@ def logger
     end
   end
 
-  def message(msg)
+  def message(msg, num=1, verbose=false)
```

Comment:
- let's use the name `n` to be consistent across the stack
- let's init params to `nil` to be consistent with the Python code (https://github.com/wit-ai/pywit/pull/100)
- `params[:n] = num if not num.nil?`
- `params[:verbose] = num if not verbose.nil?`